index.md

```diff
@@ -1,12 +1,17 @@
 ---
 layout: home
-conference_ad_row:
-  title: Spinnaker Summit 2019
-  excerpt: "Spinnaker Summit 2019 took place in San Diego, CA from November 15-17th. Hundreds of people from the Spinnaker community gathered and shared stories about their experiences with Spinnaker and Continuous Delivery. See the insights from Spinnaker contributors and users in [full session recordings here](https://www.youtube.com/playlist?list=PL4yLrwUObNkvGzXKSEU8miQTlvZyZvXWq)."
-  image_path: assets/images/summit-socialmedia-transparent.png
-  alt: "Spinnaker Summit Logo"
+promo_row:
+  title: Sign Up for Virtual Spinnaker Gardening Days
+  excerpt: "Have fun and move the needle on continuous delivery at Spinnaker Gardening Days #communityHack, sponsored by Salesforce! [Register](https://go.armory.io/gardening) and join Spinnaker users and companies on April 9 & 10 to learn and build. Enjoy a collaborative hackathon, free training, and community celebration. We’ll dig into open-source issues and features as well as extend the ecosystem. [Click to learn more](https://github.com/spinnaker-hackathon/gardening)."
+  image_path: assets/images/gardening.png
+  alt: "Spinnaker Gardening Days Logo"
+slack_row:
+  title: Join Spinnaker Slack
+  excerpt: "Visit [join.spinnaker.io](https://go.armory.io/spinslack) to get help on Spinnaker Slack, a place for the community to come together. Use this vibrant workspace to ask and answer questions, connect with other operators and users, discuss issues with SIGs, and learn Spinnaker!"
```

Comment:
Hm, might be better to link directly to join.spinnaker.io rather than to an Armory url that redirects there?

Reply:
@dorbin understood, here's my commit message on this:

>  I received a request that we be able to track how many folks join slack using the link from spinnaker.io. The easiest way for me to do this currently is to use Armory's existing rebrandly account which includes detailed link tracking, so I've added that URL here. I'm sure there is a way to do this with Google Analytics and Tag Manager, although I don't know it off the top of my head. I want to know, are we comfortable with me using the rebrandly approach if we're not using it for anything other than tracking? Or do I need to look into an alternative approach? Thanks!

I'd love your thoughts @dorbin @brian-armory !